src/content/docs/reference/custom-metrics/ads.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Ads custom metric
+description: Reference docs for the feature struct
+---
+
+_Appears in: [`custom_metrics` struct](/reference/structs/custom-metrics/)_\
+_As: [`ads`](/reference/structs/custom-metrics/#ads)_
+
+## Schema
+
+| Field name                                | Type    | Description                                                                                           |
+|-------------------------------------------|---------|-------------------------------------------------------------------------------------------------------|
+| `ads`                                     | record  | Contains information about the ads.txt file.                                                          |
+| `ads.present`                             | boolean | Indicates if the ads.txt file is present.                                                             |
+| `ads.status`                              | integer | HTTP status code of the ads.txt file response.                                                        |
+| `ads.redirected`                          | boolean | Indicates if the ads.txt file request was redirected.                                                 |
+| `ads.redirected_to`                       | string  | URL to which the ads.txt resource was redirected.                                                     |
+| `ads.account_count`                       | integer | Number of advertising accounts listed in the ads.txt file.                                            |
+| `ads.account_types`                       | record  | Types of accounts (direct or reseller) listed in the ads.txt file.                                    |
+| `ads.account_types.direct`                | record  | Information about direct advertising accounts.                                                        |
+| `ads.account_types.direct.domains`        | array   | List of domains with direct advertising accounts.                                                     |
```

Comment:
Optional suggestion: specifying the type contained in the array, eg `Array<string>`

You do this once in the privacy doc, and it'd be good to make it consistent either way